build.gradle.kts

```diff
@@ -27,6 +27,7 @@ configurations.all {
                 "com.fasterxml.jackson.core:jackson-core" -> useVersion("2.9.4")
                 "org.slf4j:slf4j-api" -> useVersion("1.8.0-alpha2")
                 "org.apache.httpcomponents:httpcore" -> useVersion("4.4.9")
+                "commons-codec:commons-codec" -> useVersion("1.11")
```

Comment:
A new dependency? What is it for?

Reply:
Ok, it's an old one. Just a version bump.

Reply:
Old dependency, new version conflict. That's what happens if libs don't use version ranges: uncountable bogus version conflicts.